lib/matplotlib/ticker.py

```diff
@@ -242,6 +245,9 @@ def set_view_interval(self, vmin, vmax):
     def get_minpos(self):
         return self._minpos
 
+    def set_minpos(self, minpos=0):
```

Comment:
This is a module private class that attempts to mimic a couple of the behaviors of `Axis` that are used by formatters and locators. I think that ` get_minpos` is part of that API and shouldn't be touched. I also think that turning `minpos` into a property would cause issues for the same reason. `set_minpos` is a thing I introduced to avoid reallocating a new `_DummyAxis` every time `_minpos` changes because of a transformer change.